diff --git a/config/sl_iostream_usart_vcom_config.h b/config/sl_iostream_usart_vcom_config.h
index 330844c..4721068 100644
--- a/config/sl_iostream_usart_vcom_config.h
+++ b/config/sl_iostream_usart_vcom_config.h
@@ -61,7 +61,7 @@
 // <usartHwFlowControlCtsAndRts=> CTS/RTS
 // <uartFlowControlSoftware=> Software Flow control (XON/XOFF)
 // <i> Default: usartHwFlowControlNone
-#define SL_IOSTREAM_USART_VCOM_FLOW_CONTROL_TYPE     usartHwFlowControlCtsAndRts
+#define SL_IOSTREAM_USART_VCOM_FLOW_CONTROL_TYPE     uartFlowControlSoftware
 
 // <o SL_IOSTREAM_USART_VCOM_RX_BUFFER_SIZE> Receive buffer size
 // <i> Default: 32
@@ -84,15 +84,28 @@
 // <<< sl:start pin_tool >>>
 // <usart signal=TX,RX,(CTS),(RTS)> SL_IOSTREAM_USART_VCOM
 // $[USART_SL_IOSTREAM_USART_VCOM]
-#warning "IO Stream USART peripheral not configured"
-// #define SL_IOSTREAM_USART_VCOM_PERIPHERAL      USART0
-// #define SL_IOSTREAM_USART_VCOM_PERIPHERAL_NO   0
+#ifndef SL_IOSTREAM_USART_VCOM_PERIPHERAL       
+#define SL_IOSTREAM_USART_VCOM_PERIPHERAL        USART0
+#endif
+#ifndef SL_IOSTREAM_USART_VCOM_PERIPHERAL_NO    
+#define SL_IOSTREAM_USART_VCOM_PERIPHERAL_NO     0
+#endif
 
-// #define SL_IOSTREAM_USART_VCOM_TX_PORT         gpioPortC
-// #define SL_IOSTREAM_USART_VCOM_TX_PIN          11
+// USART0 TX on PC00
+#ifndef SL_IOSTREAM_USART_VCOM_TX_PORT          
+#define SL_IOSTREAM_USART_VCOM_TX_PORT           gpioPortC
+#endif
+#ifndef SL_IOSTREAM_USART_VCOM_TX_PIN           
+#define SL_IOSTREAM_USART_VCOM_TX_PIN            0
+#endif
 
-// #define SL_IOSTREAM_USART_VCOM_RX_PORT         gpioPortC
-// #define SL_IOSTREAM_USART_VCOM_RX_PIN          10
+// USART0 RX on PC01
+#ifndef SL_IOSTREAM_USART_VCOM_RX_PORT          
+#define SL_IOSTREAM_USART_VCOM_RX_PORT           gpioPortC
+#endif
+#ifndef SL_IOSTREAM_USART_VCOM_RX_PIN           
+#define SL_IOSTREAM_USART_VCOM_RX_PIN            1
+#endif
 
 // #define SL_IOSTREAM_USART_VCOM_CTS_PORT        gpioPortA
 // #define SL_IOSTREAM_USART_VCOM_CTS_PIN         12
